openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/IngestionPipelineRepository.java

```diff
@@ -190,6 +193,25 @@ public RestUtil.PutResponse<?> addPipelineStatus(UriInfo uriInfo, String fqn, Pi
     return new RestUtil.PutResponse<>(Response.Status.CREATED, changeEvent, RestUtil.ENTITY_FIELDS_CHANGED);
   }
 
+  public IngestionPipeline checkProfileSampleType(IngestionPipeline ingestionPipeline) throws IOException {
+    if (Objects.equals(ingestionPipeline.getPipelineType().value(), "profiler")) {
+      JSONObject origSourceConfig =
+          new JSONObject(JsonUtils.pojoToJson(ingestionPipeline.getSourceConfig().getConfig()));
+      ObjectMapper objMapper = new ObjectMapper();
```

Comment:
please check the JsonUtils and other code we don't need to initialize another ObjectMapper here